app/Http/Controllers/API/v1/TrustedUserController.php

```diff
@@ -0,0 +1,103 @@
+<?php
```

Comment:
I would prefer, adding the option "self" to the path variable `userId` as mentioned in Zalando's [143 exception](https://opensource.zalando.com/restful-api-guidelines/#143): 
> you may define `/employees/self` or `/employees/self/personal-details` as resource paths 

This would reduce the necessity to fetch the logged in user id.

We could document this in two ways: Either change the variable `userId` to multiple types with `oneOf` ([link](https://swagger.io/docs/specification/data-models/data-types/#string)) or document a second endpoint with the variable missing. 

What do you think?

Reply:
done, sounds good